nexis/lib/widgets/auth/custom_column.dart

```diff
@@ -0,0 +1,192 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:nexis/widgets/auth/custom_input_field.dart';
+import '../custom_button.dart';
+import 'column_type.dart';
+import 'size_helper.dart';
+
+class CustomColumn extends StatelessWidget {
+  final ColumnType type;
+  final String? largeLabel;
+  final String? mediumLabel;
+  final String? mediumBody;
+  final String? smallLabel;
+  final TextEditingController? controller;
+  final void Function(String)? onSubmitted;
+  final GestureRecognizer? recognizer;
+  final bool? obscureText;
+  final String? hint;
+  final void Function()? onTap;
+  final void Function()? onPressed;
+  final String? buttonText;
+  final TextEditingController? monthController;
+  final TextEditingController? dayController;
+  final TextEditingController? yearController;
+  final bool? readOnly;
+
+  const CustomColumn(
```

Comment:
implement super.key like this:
`
const CustomColumn(
    Key? key,
    this.type,
    {this.largeLabel,
    this.mediumLabel,
    this.mediumBody,
    this.smallLabel,
    this.controller,
    this.onSubmitted,
    this.recognizer,
    this.obscureText,
    this.hint,
    this.onTap,
    this.onPressed,
    this.buttonText,
    this.monthController,
    this.dayController,
    this.yearController,
    this.readOnly,
    }) : super(key: key);
`